integrationTests/state/stateTrie_test.go

```diff
@@ -1069,3 +1068,96 @@ func BenchmarkTxExecution(b *testing.B) {
 		integrationTests.AdbEmulateBalanceTxSafeExecution(acntSrc.(*state.Account), acntDest.(*state.Account), adb, big.NewInt(1))
 	}
 }
+
```

Comment:
check warnings on file, shadowing variable err can be fixed.

Reply:
Done.